fmriprep/workflows/anatomical.py

```diff
@@ -75,10 +79,46 @@ def t1w_preprocessing(name='t1w_preprocessing', settings=None):
         ),
         name='T1_2_MNI_Registration'
     )
-    # should not be necesssary byt does not hurt - make sure the multiproc
+    # should not be necesssary but does not hurt - make sure the multiproc
     # scheduler knows the resource limits
     t1_2_mni.interface.num_threads = settings['ants_nthreads']
 
+    # 6. FreeSurfer reconstruction
+    autorecon1 = pe.Node(
+        freesurfer.ReconAll(
+            directive='autorecon1',
+            args='-noskullstrip -parallel -openmp {:d}'.format(cpu_count())),
+        name='Reconstruction')
+    autorecon1._interface._can_resume = False
+    if 'FREESURFER_SUBJECTS' in os.environ:
+        autorecon1.inputs.subjects_dir = os.getenv('FREESURFER_SUBJECTS')
+
+    convert_asw = pe.Node(
+        freesurfer.MRIConvert(out_type='mgz'),
+        name='SkullstrippedMGZ')
+
+    def inject_skullstripped(subjects_dir, subject_id, skullstripped):
+        import os
+        from nipype.utils.filemanip import copyfile
+        mridir = os.path.join(subjects_dir, subject_id, 'mri')
+        bm_auto = os.path.join(mridir, 'brainmask.auto.mgz')
+        bm = os.path.join(mridir, 'brainmask.mgz')
+        copyfile(skullstripped, bm_auto)
+        copyfile(bm_auto, bm)
+        return subjects_dir, subject_id
+
+    injector = pe.Node(
+        niu.Function(
+            function=inject_skullstripped,
+            input_names=['subjects_dir', 'subject_id', 'skullstripped'],
+            output_names=['subjects_dir', 'subject_id']),
+        name='InjectSkullstrip')
+
+    reconall = pe.Node(
+        freesurfer.ReconAll(
+            args='-parallel -openmp {:d}'.format(cpu_count())),
+        name='Reconstruction2')
```

Comment:
The erroring bit:

```
#-----------------------------------------
#@# Parcellation Stats 2 lh Thu Jan 26 17:41:39 UTC 2017
/scratch/workflow_enumerator/01/t1w_preprocessing/Reconstruction/recon_all/scripts

 mris_anatomical_stats -th3 -mgz -cortex ../label/lh.cortex.label -f ../stats/lh.aparc.a2009s.stats -b -a ../label/lh.aparc.a2009s.annot -c ../label/aparc.annot.a2009s.ctab recon_all lh white

#-----------------------------------------
#@# Parcellation Stats 2 rh Thu Jan 26 17:41:39 UTC 2017
/scratch/workflow_enumerator/01/t1w_preprocessing/Reconstruction/recon_all/scripts

 mris_anatomical_stats -th3 -mgz -cortex ../label/rh.cortex.label -f ../stats/rh.aparc.a2009s.stats -b -a ../label/rh.aparc.a2009s.annot -c ../label/aparc.annot.a2009s.ctab recon_all rh white

Waiting for PID 509 of (509 512) to complete...
/opt/freesurfer/bin/reconbatchjobs: line 81:   509 Segmentation fault      (core dumped) exec $JOB >> $LOG 2>&1
Waiting for PID 512 of (509 512) to complete...
/opt/freesurfer/bin/reconbatchjobs: line 81:   512 Segmentation fault      (core dumped) exec $JOB >> $LOG 2>&1
PIDs (509 512) completed and logs appended.
Linux 336cda4cfeae 4.4.0-59-generic #80-Ubuntu SMP Fri Jan 6 17:47:47 UTC 2017 x86_64 x86_64 x86_64 GNU/Linux
```

I've seen `mris_anatomical_stats` segfault issues before. I suspect it's because of the huge chunk of skull left by AFNI skull stripping (which is probably because of the low-res test data).

Short term: we'll kill the `mris_anatomical_stats` outputs, at least for testing. The outputs aren't used downstream in recon-all.

Medium term: I'll try to get a minimal reproduction (and better logs) for the FreeSurfer folks. Even if it fails, it shouldn't be segfaulting.

Somewhat relevant: There's no way to actually run these on CircleCI. I'll hook up the `--no-freesurfer` option for the tests, but we should figure out a way to test and validate FreeSurfer bits (at least) before releases.